WordPress/Classes/Services/PostTagService.m

```diff
@@ -13,8 +13,8 @@ @interface PostTagService ()
 @implementation PostTagService
 
 - (void)syncTagsForBlog:(Blog *)blog
-                success:(nullable void (^)(void))success
-                failure:(nullable void (^)(NSError *error))failure
+                success:(nullable void (^)(NSArray <PostTag *> *tags))success
+                failure:(nullable void (^)(NSError *error))failure;
```

Comment:
Looks like a stray semicolon might have sneaked in here.

Reply:
Ouch. Old habits die hard... It's a little thing I starting doing years ago (if you copy the method declaration from the .h file, you ca just paste it i the .m file as it is). Sorry, I will remove it right away!